PokemonRampUp/Model/Pokemon.swift

```diff
@@ -8,27 +8,24 @@
 import Foundation
 import UIKit
 
-struct Pokemon {
+// MARK: - Pokemon
+struct Pokemon: Decodable {
     let name: String
-    let type: String
-    let hiddenAbilites: [String]
-    let image: UIImage
-    let moves: [String]
+    let types: [TypeElement]
```

Comment:
A pokemon can be more than 1 type?

Reply:
Yes, pokemon can have multiple times. [Froslass](https://pokemon.fandom.com/wiki/Froslass) is both ice and ghost

Reply:
elementTypes  - [elementType]